docs/en/userguide/01_User_manual/01_Setting_up_MFA.md

```diff
@@ -0,0 +1,51 @@
+title: Setting Up MFA
+summary: Running initial configuration of an MFA method for your account
+
+# Setting Up MFA
+
+#### 1. Prompted to setup MFA
+
+When MFA is installed on your site, the first time you log in you will be
+prompted to set it up for your account.
+
+<img src="../_images/01-01-1-mfa-setup-intro.png" width="1024" />
```

Comment:
I've outlined why I've taken this approach in the revised [User Help Image guide](https://silverstripe.atlassian.net/wiki/spaces/UX/pages/624918544/User+help+Image+update+guide). in short, I'm taking retina-quality screenshots, and then forcing the browser to render them with the correct dimensions (because the default behaviour for retina images on the web is to render them at twice the size). Standard Markdown makes no provisions for adjusting attributes of an image AFAICT, so using `img` tags is the solve.

Reply:
I thought it might’ve been something like that, sweet as